conf/conf.go

```diff
@@ -24,52 +23,54 @@ type Configuration struct {
 func readEnvRequired(varName string) string {
 	val, precent := os.LookupEnv(varName)
 	if !precent {
-		rl.Fatal(context.Background(), fmt.Sprintf("panic: Env var '%s' not set", varName))
-		panic("exiting")
+		log.Fatal(fmt.Sprintf("panic: Env var '%s' not set", varName))
 	}
 	return val
 }
 
 func readEnvFallback(varName string, fallback string) string {
-	val, precent := os.LookupEnv(varName)
-	if !precent {
+	val, present := os.LookupEnv(varName)
+	if !present {
 		return fallback
 	}
 	return val
 }
 
 func readEnvInteger(varName string, fallback int) int {
-	val, precent := os.LookupEnv(varName)
-	if !precent {
+	val, present := os.LookupEnv(varName)
+	if !present {
 		return fallback
 	}
 	num, err := strconv.Atoi(val)
 	if err != nil {
-		rl.Fatal(context.Background(), fmt.Sprintf("panic: Env var '%s' is not an integer", varName))
-		panic("exiting")
+		log.Fatal(fmt.Sprintf("panic: Env var '%s' could not be parsed as integer", varName))
 	}
 	return num
 }
 
 func readEnvBoolean(varName string, fallback bool) bool {
-	val, precent := os.LookupEnv(varName)
-	if !precent {
+	val, present := os.LookupEnv(varName)
+	if !present {
 		return fallback
 	}
-	return val == "true"
+	bool, err := strconv.ParseBool(val)
```

Comment:
Haha, my bad about the naming of `bool`. I get a warning that this collides with the built in type which is fair enough 😅